### ISSUE TITLE
Exec command weird behavior 

### ISSUE THREAD

**Issue by maqboolkhan** [shelljs/shelljs#706]
### Description of the bug: exec command weird behavior 

hi i want to run a shell script using `shell.exec`
but having weird behavior (kindly dont laugh on my node version i just cant update it)

but when i do 

```
shell.exec('./build.sh');
```

it outputs as it running this command infinite time and when i try async method 

as

```
shell.exec('./build.sh', function(code, stdout, stderr) {
		console.log('Exit code:', code);
		console.log('Program output:', stdout);
		console.log('Program stderr:', stderr);
	});
```

no output produced 
so its a very weird condtion for me kindly help thanks

**Comment by nfischer** [comment]
Hi @maqboolkhan. We dropped support for node v0.10 when we released ShellJS v0.7.0 (for the very reason you're seeing).

I strongly recommend upgrading to a more recent version, since v0.10 has been deprecated for quite some time. Take a look at [this page](https://github.com/nodejs/LTS) for an idea of which versions of NodeJS are still supported.

**Comment by maqboolkhan** [comment]
thank you @nfischer i understand thats why i close this question thanks man for your co operation
